src/signature_accumulator.rs

```diff
@@ -143,7 +135,7 @@ mod tests {
 
     struct Env {
         _msg_sender_id: FullId,
-        senders: BTreeSet<PublicId>,
+        _senders: BTreeSet<PublicId>,
```

Comment:
I take it `_msg_sender_id` and `_senders` are unused (as they begin with an underscore)? If yes, shouldn't we just remove these fields?

Reply:
`_msg_sender_id` was there previously, fo I thought it is kept for some `recording` purpose, hence leave the `_senders` as well.
I am ok to remove both of them